draft-ietf-rtcweb-jsep.xml

```diff
@@ -2621,19 +2621,28 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
                     only included if such an option was present in the
                     offer.</t>
 
-                    <t>The next step is to generate lip sync groups as
-                    defined in <xref target="RFC5888" />, Section 7. For
-                    each MediaStream with more than one referenced
-                    RtpTransceiver, a group of type "LS" MUST be added
-                    that contains the mid values for each RtpTransceiver
-                    added with that MediaStream. In some cases this may
-                    result in adding a mid to a given LS group that was
-                    not in that LS group in the associated
-                    offer. Although this is not allowed by <xref
-                    target="RFC5888" />, it is allowed when implementing
-                    this specification. [[OPEN ISSUE: This is still
-                    under discussion. See:
-                    https://github.com/rtcweb-wg/jsep/issues/162.]]</t>
+                    <t>The next step is to generate session-level lip
+                    sync groups as defined in <xref target="RFC5888"/>,
+                    Section 7. All Tracks in the same MediaStream need
+                    to be placed in the same "LS" group as long as
+                    theses tracks were in the offered to by synchronized
+                    in the offer.  RFC5888 only allows the groups in the
+                    answer to be a subset of what was in the offer. If
+                    there are tracks that the same MediaStream that can
+                    not be added to the "LS" group, once the state moves
+                    back to "stable" the browser MAY initiate another
+                    offer/answer negotiation put all the tracks in the
+                    MediaStream into the "LS" group or they can be left
+                    in a state that though answer wishes them to be
+                    synchronized, the answer can not signal them as
```

Comment:
This paragraph is hard to parse. The paragraph seems to say the same thing twice in two different ways, but the language is different enough that I can't be sure there's not a subtle difference.

Suggested replacement for the whole paragraph starting at "If there are tracks... if it is critical for their use case": "If there are tracks that belong to a MediaStream that cannot be added to the "LS" group defined by the offerer, the answerer MAY add them without synchronization. It MAY also skip adding the tracks to the answer, and instead create a new offer after sending the answer."